lib/v2/screens/explore_screens/explore/explore_screen.dart

```diff
@@ -27,114 +27,115 @@ class ExploreScreen extends StatelessWidget {
       create: (_) => ExploreBloc()..add(const LoadExploreData()),
       child: Scaffold(
         appBar: AppBar(title: Text('Explore'.i18n)),
-        body: BlocListener<ExploreBloc, ExploreState>(
-          listenWhen: (_, current) => current.pageCommand != null,
-          listener: (context, state) {
-            var pageCommand = state.pageCommand;
-            BlocProvider.of<ExploreBloc>(context)..add(const ClearExplorePageCommand());
-            if (pageCommand is NavigateToRoute) {
-              NavigationService.of(context).navigateTo(pageCommand.route);
-            }
-            if (pageCommand is ShowErrorMessage) {
-              SnackBarInfo(pageCommand.message, ScaffoldMessenger.of(context)).show();
-            }
-          },
-          child: Stack(
-            children: [
-              Container(
-                height: height,
-                child: ListView(
-                  padding: const EdgeInsets.all(horizontalEdgePadding),
-                  children: [
-                    Row(
+        body: BlocConsumer<ExploreBloc, ExploreState>(
```

Comment:
BlocConsumer<ExploreBloc, ExploreState>(

This is the real change here

Reply:
I do not understand why the Bloc Consumer if there is nothing to be rebuild on this screen besides the DHO card.

Reply:
Because you need access to the bloc in order to do the navigation. 
If you dont use a consumer, you dont have access to the bloc in order to call 

Bloc.of(contetxt)

Reply:
oh .. I see this was not recovering the proper context in that widget tree configuration.

Sorry guys I did not think it will affect in that aspect.

Reply:
Well that's why I hadn't merged that PR lacks a review 👀